plugins/bazaar-backend/package.json

```diff
@@ -6,10 +6,23 @@
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public",
-    "main": "dist/index.cjs.js",
-    "types": "dist/index.d.ts",
     "alphaTypes": "dist/index.alpha.d.ts"
```

Comment:
But this one stays?

Reply:
Yep just missed removing `alphaTypes` here